storage/storage.go

```diff
@@ -268,7 +268,7 @@ const (
 type URLStyle interface {
```

Comment:
Can you also update the docs for PathStyle to indicate that the provided hostname is taken into account?

Reply:
Done. To-do on virtual hosted style, if we decide to support it.